deploy/build.sh

```diff
@@ -417,7 +423,7 @@ fi
 #
 if [ -r "${SRCDIR}/deploy/os/${OS}.opts" ]
 then
-    os_opts=$(cat ${SRCDIR}/deploy/os/${OS}.opts)
+    os_opts=$(fgrep -v "#" ${SRCDIR}/deploy/os/${OS}.opts)
```

Comment:
Good question (and a good catch).   Every system I've worked on has had fgrep.   However, it would be safer to use grep.   Especially because the man page (on Ubuntu 20) indicates that egrep, fgrep and rgrep are now deprecated.

I will make the change, test it, and then resubmit.